MetaLamp/lending-pool/client/package.json

```diff
@@ -26,8 +26,10 @@
     "jquery": "^3.3.1",
     "json-bigint": "^1.0.0",
     "node-sass": "^4.12.0",
+    "purescript": "^0.13.6",
```

Comment:
Is there a need to specify `purescript` and `spago` as local dependencies? Aren't they available via nix-shell already? I think this causes `npm install` to build `spago`, `purescript` and `ghc`. Also it fails for me.

![Screenshot from 2021-06-28 16-16-12](https://user-images.githubusercontent.com/43477987/123614191-4d2fb700-d82e-11eb-897b-69df2db55f96.png)